force-app/main/default/classes/DonationHistoryController.cls

```diff
@@ -73,11 +77,37 @@ public with sharing class DonationHistoryController {
             throw new AuraHandledException(ex.getMessage());
         }
     }
+    
+    /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description method to return whether or not Payments are enabled in org
+    * @return boolean isPaymentEnabled
+    */
+    @AuraEnabled(cacheable=true)
+    public static boolean arePaymentsEnabled(){
+        npe01__Contacts_And_Orgs_Settings__c settings = npe01__Contacts_And_Orgs_Settings__c.getOrgDefaults();
+		boolean isPaymentEnabled = settings.npe01__Payments_Enabled__c;
+		return isPaymentEnabled;
+    }
 
-    private static List<DonationHistoryView> buildHistoryFromOCRs(List<Opportunity> contactRoles) {
+    /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description method creates list of DonationHistoryView to return to LWC
+    * @return List<DonationHistoryView>
+    */
+    private static List<DonationHistoryView> buildHistoryFromOCRs( List<Opportunity> contactRoles) {
```

Comment:
Do we still have contactRoles? I think opportunities is a better name to this variable

Reply:
agreed